tests/integration/__init__.py

```diff
@@ -819,6 +823,38 @@ def setup_keyspace(ipformat=None, wait=True, protocol_version=None, port=9042):
         cluster.shutdown()
 
 
+def is_scylla_enterprise(version: Version) -> bool:
+    return version > Version('2000.1.1')
+
+
+def xfail_scylla_version_lt(reason, oss_scylla_version, ent_scylla_version, *args, **kwargs):
+    """
+    It is used to mark tests that are going to fail on certain scylla versions.
+    :param reason: message to fail test with
+    :param oss_scylla_version: str, oss version from which test supposed to succeed
+    :param ent_scylla_version: str, enterprise version from which test supposed to succeed. It should end with `.1.1`
+    """
+    if not reason.startswith("scylladb/scylladb#"):
+        raise ValueError('reason should start with scylladb/scylladb#<issue-id> to reference issue in scylla repo')
+
+    if not isinstance(ent_scylla_version, str):
+        raise ValueError('ent_scylla_version should be a str')
+
+    if not ent_scylla_version.endswith("1.1"):
+        raise ValueError('ent_scylla_version should end with "1.1"')
+
+    if SCYLLA_VERSION is None:
+        return pytest.mark.skipif(False, reason="It is just a NoOP Decor, should not skip anything")
+
+    current_version = Version(get_scylla_version(SCYLLA_VERSION))
+
+    if is_scylla_enterprise(current_version):
+        return pytest.mark.xfail(current_version < Version(ent_scylla_version),
+                                 reason=reason, *args, **kwargs)
+
+    return pytest.mark.xfail(current_version < Version(oss_scylla_version), reason=reason, *args, **kwargs)
+
+
```

Comment:
Why did you add this in this PR? It is unused.

Reply:
Sorry, I did not expected you to review it that quick, it was not ready, not it is used, please check it.